src/libmeasurement_kit/ooni/web_connectivity.cpp

```diff
@@ -583,12 +583,6 @@ void web_connectivity(std::string input, Settings options,
     (*entry)["tcp_connect"] = nlohmann::json::array();
     (*entry)["control"] = nlohmann::json({});
 
-    if (!mk::startswith(input, "http://") &&
-        !mk::startswith(input, "https://")) {
-        // Similarly to ooni-probe also accept a list of endpoints
-        input = "http://" + input;
-    }
-
```

Comment:
Let's remember to mention this somewhere in the changelog or in a prominent location so we don't forget.

Reply:
https://github.com/measurement-kit/measurement-kit/issues/1882